test/e2e_node/e2e_service.go

```diff
@@ -384,7 +384,8 @@ func (es *e2eService) startKubeletServer() (*server, error) {
 		}
 		cmdArgs = append(cmdArgs,
 			"--network-plugin=kubenet",
-			"--network-plugin-dir", filepath.Join(cwd, CNIDirectory, "bin")) // Enable kubenet
+			// TODO(random-liu): Deal with the cni directory better.
```

Comment:
Will update the comment to make it more clear. I just mean that we should use the same cni directory as `remote/remote.go`